web/components/molecules/DesktopNavBar.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
```

Comment:
It seems like there's a lot of overlap between the code in DesktopNavBar and MobileNavBar, so they should be combined into one component where styles for only the Desktop version are added with `sm:`. This will make it easier to make updates as well! Let me know if I'm mistaken about the amount of overlap/ease of combination though 😅